workflow/infrastructure-monitoring/container-monitoring.mdx

```diff
@@ -1,73 +1,131 @@
 ---
-title: "Container Monitoring"
+title: Infrastructure Monitoring
 ---
-Overview
---------
-Container Monitoring provides real-time visibility into health and performance of containerized environments. 
 
-Container List
---------------
-The container list enables visibility into all containers across your environment excluding Kubernetes. It provides a detailed overview of your containers' health, resource consumption, and deployment. The collection of containers is container id, container name, host, status, cpu usage, ram usage, etc.
+This guide provides an overview of Middleware’s Infrastructure Monitoring page. Effortlessly collect, configure, and visualize complex telemetry data with our autodetection capabilities across your entire Infrastructure stack.
 
-![](/images/infra-monitoring/c1.png)
+Quickly isolate and drill into blockers within your teams Infrastructure stack, monitor your cluster, node, and pod data in one place, and effortlessly manage and review your overall Infrastructure utilization cluster by cluster.
```

Comment:
This is a rephrasing of the paragraph above. Let's use this space to explain _how_ our infra monitoring works and use it as an opportunity to link back to the infra install pages. E.g.

"The Middleware Host Agent auto collects data across the stack using the Otel framework. While collecting the Agent will also compress and batch the data to your account to allow near real time analysis on all data points regarding your clusters, nodes, pods and services. 

Middleware supports infra monitoring on many distributions, all can be found [here](some link)."